src/inputs/RoleEditor/RoleEditor.js

```diff
@@ -37,7 +37,7 @@ export const RoleEditor = ({
   const classes = useStyles();
   const avatar = icon ? <Avatar>{icon}</Avatar> : <DefaultAvatar userName={agentName} />;
   return (
-    <Grid data-cy={`role-editor-${agentName.replaceAll(/[@.]/g, '')}`} item xs={12} className={classes.rolesEditor}>
+    <Grid data-cy={`role-editor-${agentName.replace(/[@.]/g, '')}`} item xs={12} className={classes.rolesEditor}>
```

Comment:
@nborde-CSM could you make the same change on your branch, in the factorized function ? Thanks !